spec/worker_spec.rb

```diff
@@ -45,6 +45,49 @@
         expect(job).to_not be_nil
         expect(job['last_error']).to_not be_nil
       end
+
+      context 'for a non-QueueClassicPlus job' do
+        after { Object.send(:remove_const, :NonQcJob) }
```

Comment:
IIUC this is dynamically adding / removing a class so that we can test QC execution without fear that we're going to clash with other gems.

I think this needs a comment about what's going on, or just use a name under the gem's name space? (`QueueClassicPlus::Worker::Test::NoQcJob` or something?)

Maybe just create the class as normal?
```
class QueueClassicPlus::Worker::Test::NoQcJob < ActiveJob::Base
  ...
end
```

Reply:
The issue with creating the class as normal is that `ActiveJob` isn't always available. Removing the constant is out of FUD that things will break if it's still present if `ActiveJob` isn't—but `ActiveJob` is never removed after it's first required, so I guess this isn't needed?